codegenerator/cli/src/config_parsing/system_config.rs

```diff
@@ -997,33 +1043,6 @@ impl FieldSelection {
     }
 }
 
-#[derive(Debug, Clone, PartialEq)]
-struct NormalizedEthAbiEvent(EthAbiEvent);
```

Comment:
I see you've removed these type semantics. Just wanted to make you aware of the new type idiom in rust if you're not familiar: https://doc.rust-lang.org/rust-by-example/generics/new_types.html

It's useful in the sense that it shows when I use this type elsewhere that it's been parsed/normalized.